README.md

```diff
@@ -0,0 +1,56 @@
+# Deploying Encrypted Container Images with Cloud API Adatpor
+
+## Encrypt Image
+
+### Generate Image Encryption / Decryption Symmetric Key
+
+```bash
+export KEY_FILE="$(pwd)/key.bin"
+head -c 32 /dev/urandom | openssl enc >"$KEY_FILE"
```

Comment:
You can add a note here stating that the key needs to be 32 bytes. Afaik, this restriction is still there with KBS